root/release/merge.tt

```diff
@@ -63,12 +61,18 @@
           </p>
           <table class="tbl">
             <tbody>
+              <style>
+                th span.error { margin: 0 12px 0 6px; }
+              </style>
               [% FOR medium=mediums %]
               [% field = form.field('medium_positions').field('map').field(loop.index) %]
               <tr class="subh">
                 <th colspan="4">
                   <label>[% l('New position:') %]</label>
                   [% r.text(field.field('position'), size=2) %]
+                  [%~ IF field.field('position').has_errors;
+                        '<span class="error">' _ html_escape(field.field('position').errors.0) _ '</span>';
```

Comment:
Yeah, `errors` is an array of error strings for the field: http://search.cpan.org/dist/HTML-FormHandler-0.40058/lib/HTML/FormHandler/Manual/Errors.pod

In TT syntax `.errors.0` means `->errors->[0]`, so this only displays the first error if there are multiple. But that should only happen if they enter a 0 or negative position multiple times...